packages/services/src/types/index.ts

```diff
@@ -0,0 +1,1200 @@
+import { z } from "zod/v4";
```

Comment:
didn't actually review this file, I'm just assuming the types are strict code movement from the old types file in the `api` package.  If anything here actually changed please lmk

Reply:
Only additions were the following:

```
/**
 * Custom error class with HTTP status code
 */
export class ApiError extends Error {
  statusCode: number;

  constructor(statusCode: number, message: string) {
    super(message);
    this.statusCode = statusCode;
    this.name = this.constructor.name;
    Error.captureStackTrace(this, this.constructor);
  }
}

export type SpecificChainBalances = Partial<
  Record<SpecificChain, Record<string, number>>
>;

export type SpecificChainTokens = Partial<
  Record<SpecificChain, Record<string, string>>
>;
```
This moved the ApiError type from some middleware error handler file in the api app to this shared location, and adds two types to represent some data structures used in the global config object.